ydb/docs/en/core/integrations/ingestion/index.md

```diff
@@ -0,0 +1,7 @@
+# Data Ingestion {#ingestion}
```

Comment:
I don't immediately see the reason for it, but this `{#ingestion}` is getting rendered when it shouldn't:
<img width="949" alt="Screenshot 2024-09-09 at 11 51 56" src="https://github.com/user-attachments/assets/4a3d39b9-014f-47c4-b0ee-0516ea2a1ef7">

Maybe anchors are not supposed to be on `<h1>` /  `#`?